src/main.cpp

```diff
@@ -40,7 +40,7 @@
 #endif
 #include "util.h"
 #ifdef ANDROID
-#    include <QtAndroidExtras/QtAndroid>
+#    include <QtCore/private/qandroidextras_p.h>
```

Comment:
`QtCore/private/qandroidextras_p.h` I'd rather not be including "private" header files.  There is a standardised Qt6 permissions API, as far as I know -- if we're using Qt6, we should make all the necessary changes not hack the code...

So this is a blocker we'll need to address if Qt6 is required for Google Store.